Jenkinsfile

```diff
@@ -0,0 +1,128 @@
+                            returnStdout: true
+                    ).trim()
+
+                    sh "curl -v -H \"Authorization: token ${GIT_TOKEN}\" https://api.github.com/repos/${git_project_user}/${git_project}/releases/tags/v${TAG_VERSION} > ~/tag_version"
+
+                    AUTO_TAG = sh(
+                            script: "cat ~/tag_version | python -c 'import json,sys;obj=json.load(sys.stdin);print obj[\"body\"]'",
+                            returnStdout: true
+                    ).trim()
+
+                    PUBLISHED_BEFORE = sh(
+                            script: "tag_published_at=\$(cat ~/tag_version | python -c 'import json,sys;obj=json.load(sys.stdin);print obj[\"published_at\"]'); SECONDS=\$(expr \$(date +%s) - \$(date -d \"\$tag_published_at\" +%s)); expr \$SECONDS / 60 + 1",
+                            returnStdout: true
+                    ).trim()
+
+                    echo "$AUTO_TAG"
+                    echo "$TAG_VERSION"
+                    echo "$PUBLISHED_BEFORE"
+                }
+            }
+
+            if ( TAG_VERSION && PUBLISHED_BEFORE < 240 ) {
+                stage('prepare sources') {
+                    container('jnlp') {
+                        sh """
+                            cd ${BUILD_FOLDER}
+                            git clone https://${GIT_USERNAME}:${GIT_PASSWORD}@github.com/${git_project_user}/${git_project}.git src/github.com/v3io/${git_project}
+                        """
+                    }
+                }
+
+                stage('build in dood') {
+                    container('docker-cmd') {
+                        sh """
+                            cd ${BUILD_FOLDER}/src/github.com/v3io/${git_project}/netops_demo/golang/src/github.com/v3io/demos
+                            docker build . --tag netops-demo-golang:latest --tag ${docker_user}/netops-demo-golang:${TAG_VERSION} --build-arg NUCLIO_BUILD_OFFLINE=true --build-arg NUCLIO_BUILD_IMAGE_HANDLER_DIR=github.com/v3io/demos
+
+                            cd ${BUILD_FOLDER}/src/github.com/v3io/${git_project}/netops_demo/py
+                            docker build . --tag netops-demo-py:latest --tag ${docker_user}/netops-demo-py:${TAG_VERSION}
+                        """
+                    }
+                }
+
+                stage('push to hub') {
+                    container('docker-cmd') {
+                        withDockerRegistry([credentialsId: docker_credentials, url: ""]) {
+                            sh "docker push ${docker_user}/netops-demo-golang:${TAG_VERSION}"
+                            sh "docker push ${docker_user}/netops-demo-py:${TAG_VERSION}"
+                        }
+                    }
+                }
+            } else {
+                stage('warning') {
+                    if (PUBLISHED_BEFORE >= 240) {
+                        echo "Tag too old, published before $PUBLISHED_BEFORE minutes."
```

Comment:
are you sure you want a warning that no one will see instead of failing the job
or atleast mark the stage skipped - you can use the conditional_Stage in pipelinex